examples/consumer/src/main/java/io/strimzi/examples/consumer/ExampleConsumer.java

```diff
@@ -75,6 +75,16 @@ public static void main(String[] args) {
         }
     }
 
+    @SuppressWarnings("deprecation")
+    private static boolean isAccessTokenJwt(Config config) {
+        String legacy = config.getValue(Config.OAUTH_TOKENS_NOT_JWT);
+        if (legacy != null) {
+            System.out.println("[WARN] OAUTH_TOKENS_NOT_JWT is deprecated. Use OAUTH_ACCESS_TOKEN_IS_JWT (with reverse meaning) instead.");
+        }
+        return legacy != null ? !Config.isTrue(legacy) :
```

Comment:
This is just the ExampleConsumer CLI client - not something anyone will use for more than learning, how to use it on the client. Validation performed by it is more for the user experience than for validation. 
Actual validation happens later in the workflow. If both old and new options are present, then the exception is thrown, because it's treated as a misconfiguration.
It happens in [OAuth client library](https://github.com/mstruk/strimzi-kafka-oauth/blob/backport-config/oauth-client/src/main/java/io/strimzi/kafka/oauth/client/JaasClientOauthLoginCallbackHandler.java#L103) that actually handles this config under the hood and [throws an exception](https://github.com/mstruk/strimzi-kafka-oauth/blob/backport-config/oauth-common/src/main/java/io/strimzi/kafka/oauth/common/DeprecationUtil.java#L17). 

So, if both options are present then the exception will be thrown, just a little later in the code flow.